challenge_7/python/sarcodian/src/challenge_7.py

```diff
@@ -0,0 +1,12 @@
+def missing_int(L):
+
+    sum_n = 0
+    sum_n_less_1 = 0
+
+    for i in range(len(L)+1):
+        sum_n += i
```

Comment:
Thanks, can you provide a link or something that provides a deeper explanation of <i for i in range(len(L)+1)> syntax. I have seen it a lot in other code as well, but I don't get the logic behind how it works so I am wary of just using it. Or even if you provide the name for what that type of syntax is called I can google it.